docs/architecture/decisions/0002-use-core-and-security-approved-contributed-modules-when-possible.md

```diff
@@ -0,0 +1,27 @@
+# 1. Use core and security-approved contributed modules when possible
+
+## Status
+
+Proposed
+
+## Context
+HSDP has a mix of custom modules, core modules, and contributed modules. 
+
+Custom solutions have offered us the flexibility to tailor functionality to specific needs and requests; however, they can introduce potential challenges, including increased maintenance complexity and associated security considerations.
+
+Core and security-approved contributed modules and related plugins, allow us to leverage community support and regular updates to enhance maintainability and project stability.
+
+## Decision
+We have decided to use core Drupal modules and security-approved contributed modules whenever possible, opting for custom development only when no suitable core or contributed options are available. This approach is intended to ensure better maintainability, minimize risks associated with custom code, and align with best practices within the Drupal community.
+
+## Consequences
+
+- **Positive:**
+    - **Improved Maintainability:** Core and security-approved contributed modules generally receive regular updates and are better supported, reducing the burden on our development teams.
```

Comment:
Specifically when it comes to major version updates, we can rely on the upgrades being crowdsourced (and contribute where possible).